tests/core/test_boolean_serializer.py

```diff
@@ -0,0 +1,92 @@
+import pytest
+
+from ssz import (
+    DeserializationError,
+    SerializationError,
+    decode,
+    encode,
+)
+from ssz.sedes import (
+    Boolean,
+)
+
+
+@pytest.mark.parametrize(
+    'value,expected',
+    (
+        (True, b'\x01'),
+        (False, b'\x00'),
+    ),
+)
+def test_boolean_serialize_values(value, expected):
+    sedes = Boolean()
+    assert sedes.serialize(value) == expected
+
+
+@pytest.mark.parametrize(
+    'value',
+    (
+        None,
+        1,
+        0,
+        'True',
+        b'True',
+    ),
+)
+def test_boolean_serialize_bad_values(value):
+    sedes = Boolean()
+    with pytest.raises(SerializationError):
+        sedes.serialize(value)
+
+
+@pytest.mark.parametrize(
+    'value,expected',
+    (
+        (b'\x01', True),
+        (b'\x00', False),
+    ),
+)
+def test_boolean_deserialization(value, expected):
+    sedes = Boolean()
+    assert sedes.deserialize(value) == expected
+
+
+@pytest.mark.parametrize(
+    'value',
+    (
+        b' ',
```

Comment:
It'd be nice to add empty string to this list as well as the text strings `'\x00'` an `'\x01'` (as opposed to the bytestrings).